app/js/controllers/main.js

```diff
@@ -50,22 +48,34 @@ export default class MainController extends Controller {
       return;
     }
 
-    this.view.customizer.setRootNode(document.documentElement);
+    // Insert the customizer view into the app's document
     document.body.appendChild(this.view.el);
 
-    var padding = this.view.el.querySelector('fxos-customizer').offsetHeight;
-    this._styleEl = document.createElement('style');
-    this._styleEl.textContent = `
-    html, body {
-      overflow: initial !important;
-    }
+    // And display it. This has to be in a setTimeout or the CSS animation
+    // won't work correctly since we just added it above.
+    setTimeout(() => { this.view.show(); });
+
+    // Wait a bit before populating it and defining the new stylesheet
+    // so that we can just get it on the screen quickly and don't interfere
+    // with the animation
+    setTimeout(() => {
+      this.view.customizer.setRootNode(document.documentElement);
+
+      var padding = this.view.el.querySelector('fxos-customizer').offsetHeight;
+      this._styleEl = document.createElement('style');
+      this._styleEl.textContent = `
+      html, body {
+        overflow: initial !important;
+      }
 
-    body {
-      padding-bottom: ${padding}px !important;
-    }`;
-    document.head.appendChild(this._styleEl);
+      body {
+        padding-bottom: ${padding}px !important;
+      }`;
+      document.head.appendChild(this._styleEl);
 
-    this.attached = true;
+      this.attached = true;
+      this.waitToBeClosed(); // We're open, so expect a close gesture
+    }, this.view.animationLength);
```

Comment:
Yeah, that was the reason I originally wanted to have a dummy Customizer and a real one. We could animate the dummy one while rendering and painting the real one in the background. Then we could just hide the dummy one at the end of the transition. It sounds like that's not worth doing, though.